src/main.cpp

```diff
@@ -19,19 +19,18 @@ int main(int const ac, char const *const *const av)
 {
 	if (ac != 3)
 	{
-		std::cerr << "incorrect number of args\n";
+		std::cerr << "incorrect number of arguments\n";
```

Comment:
A std::endl instead of \n would be more pertinent in the case of an error.

Reply:
As you prefer, anyway, the end of the program will flush the buffer, and this end is reached at the next line.

Reply:
Yes but it would maintain a certain logic since we always put std::endl in case of an error